docs/src/_includes/components/language-switcher.html

```diff
@@ -14,9 +14,12 @@
         <select name="language selector" id="language-select" aria-describedby="language-infobox"
             class="c-custom-select switcher__select">
             {% for key, other_site in sites %}
-            <option value="{{ other_site.language.code }}" data-url="https://{{ other_site.hostname }}" {% if
-                site.language.code==other_site.language.code %} selected {% endif %}>
-                {{ other_site.language.flag }} {{ other_site.language.name }}
+            <option
+                value="{{ other_site.language.code }}"
+                data-url="https://{{ other_site.hostname }}/docs/latest{{ page.url | prettyURL }}"
+                {% if site.language.code==other_site.language.code %}selected{% endif %}
+            >
+                {{ other_site.language.flag }} {{ other_site.language.name }} ({{ other_site.footer.language_switcher.latest }})
```

Comment:
So, the idea is to always show the languages and always switch to the `latest` version in the selected language, but to add `(Latest)` to the text so that users are aware that they'll go to the `latest` version even if they're currently browsing e.g. the `HEAD` version?

Reply:
Yes, because currently HEAD always points to the English version.

Reply:
Okay, that makes sense to me.